pkgs/development/libraries/quarto/default.nix

```diff
@@ -11,14 +11,18 @@
 , makeWrapper
 , python3
 , extraPythonPackages ? ps: with ps; []
+
+# Allow overriding the Python and R used by Quarto
+, QUARTO_PYTHON ? "${python3.withPackages (ps: with ps; [ jupyter ipython ] ++ (extraPythonPackages ps))}/bin/python3"
```

Comment:
We might also want to save the constructed python wrapper in an attribute so the user can get it back out (e.g. to play around with in a devShell).

Reply:
Based on your comments and my intended use case (I need to override these environment variables to `""`), it might be best if we simply provide an *unwrapped* version of the Quarto build as a passthru argument, then people can wrap it however they want and the default wrapping does what it did before.